pwa/src/pages/documentation/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { navigate } from "gatsby";
+import * as React from "react";
+
+const IndexPage = () => {
+  navigate("/");
+  return <></>;
+};
+
+export default IndexPage;
```

Comment:
Nice catch!
I did notice that it still breaks on the organization's page breadcrumb. I suggest doing the same redirect there as you did here.